src/imp/platform/windows.rs

```diff
@@ -38,78 +34,35 @@ pub fn get_app_dir(t: AppDataType) -> Result<PathBuf, AppDirsError> {
     get_folder_path(folder_id).map(|os_str| os_str.into())
 }
 
-/// https://msdn.microsoft.com/en-us/library/dd378457.aspx#FOLDERID_RoamingAppData
-#[allow(non_upper_case_globals)]
-static FOLDERID_RoamingAppData: GUID = GUID {
-    Data1: 0x3EB685DB,
-    Data2: 0x65F9,
-    Data3: 0x4CF6,
-    Data4: [0xA0, 0x3A, 0xE3, 0xEF, 0x65, 0x72, 0x9F, 0x3D],
-};
-
-/// https://msdn.microsoft.com/en-us/library/dd378457.aspx#FOLDERID_LocalAppData
-#[allow(non_upper_case_globals)]
-static FOLDERID_LocalAppData: GUID = GUID {
-    Data1: 0xF1B32785,
-    Data2: 0x6FBA,
-    Data3: 0x4FCF,
-    Data4: [0x9D, 0x55, 0x7B, 0x8E, 0x7F, 0x15, 0x70, 0x91],
-};
-
-/// https://msdn.microsoft.com/en-us/library/dd378457.aspx#FOLDERID_ProgramData
-#[allow(non_upper_case_globals)]
-static FOLDERID_ProgramData: GUID = GUID {
-    Data1: 0x62AB5D82,
-    Data2: 0xFDC1,
-    Data3: 0x4DC3,
-    Data4: [0xA9, 0xDD, 0x07, 0x0D, 0x1D, 0x49, 0x5D, 0x97],
-};
-
-/// Wrapper around `winapi::PWSTR` to automatically free the string pointer.
+/// Wrapper around `windows::core::PWSTR` to automatically free the string pointer.
 /// This ensures the memory is freed when `get_folder_path` scope is left,
 /// regardless of whether the call succeeded or failed/panicked.
 struct SafePwstr(PWSTR);
```

Comment:
Is this struct still needed or can you use one of the new wrapper primitives from the `windows` crate?

Reply:
From what I can tell, the `windows::core::PWSTR` is not automatically deallocated as it doesn't implement `Drop`, so this struct is still needed

Reply:
There are different (allocating etc) wrapper types in `windows-strings`, not the low-level ones from `windows-core`.

Reply:
I'm okay with using own wrapper. It's just one simple call, not worth pulling in entire dependency for.

Reply:
Still, I don't see anything in `windows-strings` that could take care of this. The only types which have `Drop` implemented are `BSTR` and `HSTRING` which are different types. They also use the `SysFreeString()` call, which is different than what we need here. The only conversion between `PWSTR` and `HSTRING` is a copying one